src/cli/serve/serve.js

```diff
@@ -19,20 +19,15 @@
 
 import _ from 'lodash';
 import { statSync, lstatSync, realpathSync } from 'fs';
-import { isWorker } from 'cluster';
 import { resolve } from 'path';
 
 import { fromRoot } from '../../utils';
 import { getConfig } from '../../server/path';
-import { Config } from '../../server/config/config';
-import { readYamlConfig } from './read_yaml_config';
+import { bootstrap } from '../../core/server';
```

Comment:
__note:__ this `bootstrap` was inside of `src/core/cli/commands/serve` in my WIP CLI branch, so I had two choices here: either create something like `src/core/cli/cli.ts` that will expose `bootstrap\serve` function or create separate `src/core/server/bootstrap.ts` file. I went with the latter.